app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -9,47 +9,127 @@ class PipelineSampleReport extends React.Component {
     this.report_details = props.report_details
     this.report_page_params = props.report_page_params
     this.all_backgrounds = props.all_backgrounds;
-    this.max_rows_to_render = props.max_rows || 2000
+    this.max_rows_to_render = props.max_rows || 1000
+
     this.state = {
       taxonomy_details:  [],
-      all_genera_in_sample: [],
+      search_keys_in_sample: [],
+      lineage_map: {},
       rows_passing_filters: 0,
       rows_total: 0,
+      thresholded_taxons: [],
+      selected_taxons: [],
+      new_filter_thresholds: {
+        NT_aggregatescore: 0.0,
+        //other potential fields
+        /*
+        NT_zscore: 0.0,
+        NT_rpm: 0.0,
+        NT_r: 0.0,
+        NT_percentidentity: 0.0,
+        NT_neglogevalue: 0.0,
+        NT_percentconcordant: 0.0,
+        NR_zscore: 0.0,
+        NR_rpm: 0.0,
+        NR_r: 0.0,
+        NR_percentidentity: 0.0,
+        NR_neglogevalue: 0.0,
+        NR_percentconcordant: 0.0,
+        */
+      },
+      excluded_categories: [],
+      search_taxon_id: 0,
       loading: true
     };
 
     this.fetchReportData();
-    this.applyNewFilterThresholds = this.applyNewFilterThresholds.bind(this);
-    this.applyExcludedCategories = this.applyExcludedCategories.bind(this);
-    this.applyGenusFilter = this.applyGenusFilter.bind(this);
+    this.fetchSearchList();
```

Comment:
Typically fetching data should be done in the componentWillMount or componentDidMount lifecycle methods.  It's typically discouraged to call setState from the constructor